setup.py

```diff
@@ -216,7 +216,7 @@ def monkey_patch_THD_link_flags():
 class build_ext(setuptools.command.build_ext.build_ext):
 
     def run(self):
-        global _C_LIB, THNN, THCUNN
+        global THNN, THCUNN
```

Comment:
These global should be still unnecessary. This is only needed if you were to reassign them, and you're only modifying the attributes